frontend/packages/knative-plugin/src/components/eventing/triggers-list/TriggerListPage.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { ListPage } from '@console/internal/components/factory';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { EventingTriggerModel } from '../../../models';
+import TriggerList from './TriggerList';
+
+const TriggerListPage: React.FC<React.ComponentProps<typeof ListPage>> = (props) => (
+  <ListPage
+    canCreate={false}
+    {...props}
+    kind={referenceForModel(EventingTriggerModel)}
+    ListComponent={TriggerList}
```

Comment:
you can use the fieldSelector field here to filter our the triggers based on the broker name
```
fieldSelector={`spec.broker=${<broker name>}`}
```

Reply:
`fieldSelector` seems to have issue with `spec.broker`, so will be going with current implementation.
```Error Loading Triggers: field label not supported: spec.broker```